packages/renderer/src/lib/table/TestTable.svelte

```diff
@@ -13,21 +13,23 @@ const people: object[] = [
   { name: 'Charlie', age: 43 },
 ];
 
-const nameCol: Column<any> = new Column('Name', 'left', '3fr', TestColumnName);
+const nameCol: Column<any> = new Column('Name', { width: '3fr', renderer: TestColumnName });
 nameCol.setComparator((a, b) => a.name.localeCompare(b.name));
 
-const ageCol: Column<any> = new Column('Age', 'right', '1fr', TestColumnAge);
+const ageCol: Column<any> = new Column('Age', { align: 'right', renderer: TestColumnAge });
 ageCol.setComparator((a, b) => a.age - b.age);
 
 const columns: Column<any>[] = [nameCol, ageCol];
 
-const tableHelper = new TableHelper<any>('people', person => person.age < 50, 'People over 50 cannot be deleted');
+const row = new Row<any>();
```

Comment:
shouldn't be type to {name: string; age: string} ?

Reply:
Added Person type in latest commit.